clients/drcachesim/tracer/tracer.cpp

```diff
@@ -1609,9 +1610,12 @@ event_exit(void)
 #ifdef BUILD_PT_TRACER
     if (op_offline.get_value() && op_enable_kernel_tracing.get_value()) {
         drpttracer_exit();
-        /* Dump kcore and kallsyms to {kernel_pt_logsubdir}. */
-        if (!syscall_pt_trace_t::kernel_image_dump(kernel_pt_logsubdir)) {
-            NOTIFY(0, "WARNING: failed to run shellscript to dump kernel image\n");
+        /* Copy kcore and kallsyms to {kernel_pt_logsubdir}. */
+        std::unique_ptr<kcore_copy_t> kcore_copy(new kcore_copy_t(
```

Comment:
Looks like there is no reason to put this on the heap: may as well just have a local stack var with no need for `new` or a pointer at all, right?

Reply:
Done.